src/include/daos/drpc_modules.h

```diff
@@ -84,6 +84,8 @@ enum drpc_mgmt_method {
 	DRPC_METHOD_MGMT_GROUP_UPDATE		= 232,
 	DRPC_METHOD_MGMT_DISCONNECT		= 233,
 	DRPC_METHOD_MGMT_DEV_IDENTIFY		= 234,
+	DRPC_METHOD_MGMT_POOL_CONNECT		= 235,
+	DRPC_METHOD_MGMT_POOL_DISCONNECT	= 236,
```

Comment:
I think the names of these methods should mention monitoring. Could either call it ...MGMT_POOL_MONITOR and have an enum in the request to say what pool op it is, or call them ...MGMT_POOL_MONITOR_CONNECT/DISCONNECT.

Reply:
I added MON_ to the C defines to make them still fit in that space however I added the full string monitor to the go defines and also the C functions.